docs/developer-guide/zpool.md

```diff
@@ -0,0 +1,114 @@
+<!--
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+
+<!--
+    Copyright 2021, Joyent, Inc.
+-->
+
+# Booting the Head Node from a ZFS Pool
+
+A Triton Head Node can boot off of a ZFS pool, using its bootable filesystem
+as an on-disk equivalent to the traditional USB-key.  The `sdc-usbkey`
+command works on a disk-booting Triton Head Node nearly-identically to an
+actual USB-key-booting head node.  The only difference is that the ZFS
+bootable filesystem is case-sensitive, unlike the USB key.
+
+A ZFS pool that is bootable SHOULD be created with `zpool create -B`.  You
+can determine if a pool was created this way by querying the `bootsize`
+parameter on the pool:
+
+```
+[root@headnode ~]# zpool get bootsize zones
+NAME   PROPERTY  VALUE     SOURCE
+zones  bootsize  256M      local
+[root@headnode ~]# 
+```
+
+If a pool was not created with -B, it can still be bootable IF AND ONLY IF
+the server is bootable with BIOS. One created with -B can be bootable by
+both BIOS and EFI servers.
+
+Not all pools can be bootable.  We suggest a simple disk set for a
+ZFS-bootable Head Node.  Such sets include:
+
+- Single disk.
+- Two (mirrored) or more (raidz) same-sized disk.
+- Two or more same-sized disk with a single SSD for a log device.
+
+NOTE:  After installation a bootable pool can have log devices or log device
+mirrors added.
```

Comment:
Any reason to call out SLOG devices and not L2ARC? I assume it's also safe?

Reply:
Probably, but I've not tried that yet.  Should do so on my headnode VM.